frontend/src/__tests__/api.test.js

```diff
@@ -269,11 +435,81 @@ function positionsTests(api = { apiGET, apiPOST }) {
 // actual tests that use `apiGET` and `apiPOST`
 // eslint-disable-next-line
 function instructorsTests({ apiGET, apiPOST }) {
-    it.todo("get instructors");
-    it.todo("get instructors for session");
-    it.todo("create and delete instructor");
-    it.todo("update an instructor");
-    it.todo("add and remove instructor from position");
+    let session = null,
+        position = null,
+        instructor = null;
+    // set up a session to be available before tests run
+    beforeAll(async () => {
+        // this session will be available for all tests
+        session = await addSession({ apiGET, apiPOST });
+        position = await addPosition({ apiGET, apiPOST }, session);
+    });
+    // delete the session after the tests run
+    afterAll(async () => {
+        await deletePosition({ apiGET, apiPOST }, position);
+        await deleteSession({ apiGET, apiPOST }, session);
+    });
+
+    it("get instructors", async () => {
+        const resp = await apiGET("/instructors");
+        expect(resp).toMatchObject({ status: "success" });
+        checkPropTypes(PropTypes.arrayOf(instructorPropTypes), resp.payload);
+    });
+
+    it("get instructors for session", async () => {
+        const resp = await apiGET(`/sessions/${session.id}/instructors`);
+        expect(resp).toMatchObject({ status: "success" });
+        checkPropTypes(PropTypes.arrayOf(instructorPropTypes), resp.payload);
+    });
+
+    it("create instructor", async () => {
```

Comment:
Make this the first test so that we know something gets returned in the `get` tests.